### PR TITLE
fix document for swish, thresholded_relu, gaussian_randdom, test=document_fix

### PR DESCRIPTION
**Pull request by iclementine**
fix example code and Tensor Type to add more details.

fix typos.

图片

图片

图片

### CODE REVIEW

**Review by kuke — DISMISSED** [review]
LGTM

**Review by wanghaoshuang — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -10976,23 +10976,62 @@ def hard_sigmoid(x, slope=0.2, offset=0.5, name=None):
...
         .. code-block:: python
 
-            import paddle.fluid as fluid
-            x = fluid.layers.data(name="x", shape=[3,10,32,32], dtype="float32")
-            y = fluid.layers.swish(x, beta=2.0)
+            # declarative mode 
+            import numpy as np
+            from paddle import fluid
+   
+            x = fluid.layers.gaussian_random((2, 3), std=2., seed=10)
```
Comment (shippingwang) [inline python/paddle/fluid/layers/nn.py]: it should be ```swish```

**Review by shippingwang — CHANGES_REQUESTED** [review]

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (iclementine) [reply python/paddle/fluid/layers/nn.py]: done, it is fixed!

**Review by wanghaoshuang — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -11048,23 +11048,71 @@ def hard_sigmoid(x, slope=0.2, offset=0.5, name=None):
 @templatedoc()
 def swish(x, beta=1.0, name=None):
     """
-    ${comment}
+    Elementwise swish activation function. Se `Searching for Activation Functions <https://arxiv.org/abs/1710.05941>`_ for more details.
```
Comment (guoshengCS) [inline python/paddle/fluid/layers/nn.py]: Se->See

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -11048,23 +11048,71 @@ def hard_sigmoid(x, slope=0.2, offset=0.5, name=None):
...
+
+    .. math::
+        out = \\frac{x}{1 + e^{- beta * x}}
+    
     Args:
-        x(${x_type}): ${x_comment}
-        beta(${beta_type}|1.0): ${beta_comment}
-        name(str|None): A name for this layer(optional). If set None, the layer
-                        will be named automatically.
+        x(Variable): dtype: float32 or float64, the input.
```
Comment (guoshengCS) [inline python/paddle/fluid/layers/nn.py]: 加上Variable的类型Tensor or LoDTensor

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (iclementine) [reply python/paddle/fluid/layers/nn.py]: Done!

File: python/paddle/fluid/layers/nn.py (hunk 3 above)
Comment (iclementine) [reply python/paddle/fluid/layers/nn.py]: Done!

**Review by shippingwang — APPROVED** [review]
good job!

**Review by guoshengCS — APPROVED** [review]

**Review by kuke — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**